automol/form/_form.py

```diff
@@ -287,13 +267,12 @@ def _sort_key(entry):
     )
 
 
-def sort_vector(fml, symbs=None):
-    """Generate a sort vector for sorting various formulas against each other
+def sort_vector(fml: str, symbs: list[str] | None = None):
```

Comment:
Correction: `(fml: dict[str, int], symbs: Sequence[str] | None = None) -> tuple[int, ...]`

Sorry, the docstring led you astray (looks like some bad copy-pasting on my part). The parameter documentation should also be corrected:
```
:param fml: Stoichiometric chemical formula string
```